contracts/bridge/Bridge.sol

```diff
@@ -193,7 +193,7 @@ contract Bridge is IERC20BridgeReceiver, IERC721BridgeReceiver, Ownable, BridgeF
 
     // () requests transfer KLAY to msg.sender address on relative chain.
     function () external payable {
-        _requestKLAYTransfer(msg.sender, msg.value);
+        _requestKLAYTransfer(msg.sender, feeOfKLAY);
```

Comment:
If the user is not aware of fee, it may be confusing why the full amount does not sent. How about removing this payable method?

Reply:
() payable method is so simple and easy. But as you mentioned, the transferred amount is msg.value - fee. I would like to keep this easy way to transfer. :)